common/src/main/java/whocraft/tardis_refined/common/blockentity/console/GlobalConsoleBlockEntity.java

```diff
@@ -214,19 +214,14 @@ public void tick(Level level, BlockPos blockPos, BlockState blockState, GlobalCo
 
                 if (x.getLevel().getGameTime() % (20) == 0) {
 
-                // Check if we're crashing and if its okay to explode the TARDIS a little.
-                if (x.getPilotingManager().isCrashing() && x.getLevel().getRandom().nextInt(15) == 0) {
-                    level.explode(null, blockPos.getX(), blockPos.getY(), blockPos.getZ(), 2f, Level.ExplosionInteraction.NONE);
-                }
+                    // Check if we're crashing and if its okay to explode the TARDIS a little.
+                    if (x.getPilotingManager().isCrashing() && x.getLevel().getRandom().nextInt(15) == 0) {
+                        level.explode(null, blockPos.getX(), blockPos.getY(), blockPos.getZ(), 2f, Level.ExplosionInteraction.NONE);
+                    }
                     TardisInteriorManager intManager = x.getInteriorManager();
                     if (intManager.isCave()) {
                         intManager.setCurrentTheme(intManager.preparedTheme());
                     }
-
-//                    if (x.getLevel().getGameTime() % (20) == 0) {
-//                        serverLevel.playSound(null, blockPos, SoundEvents.NOTE_BLOCK_BELL.value(), SoundSource.BLOCKS, 10f, 2f);
```

Comment:
Not sure what this was for, I assume it was the old mini game counting down sound?

Reply:
It was a warning sound used for when the tardis entered The Danger Zone, which no longer exists.